persistence-test/src/main/java/io/stargate/it/storage/ExternalStorage.java

```diff
@@ -138,8 +139,21 @@ public void beforeTestExecution(ExtensionContext context) {
   @Override
   public void handleTestExecutionException(ExtensionContext context, Throwable throwable)
       throws Throwable {
-    LOG.warn("Error in {}", context.getUniqueId(), throwable);
-    getResource(context).ifPresent(Cluster::markError);
+    // 24-Sep-2021, tatu: When using "assume()" methods for dynamic disabling/skipping
+    //    of tests, we still get an exception -- but those are not fails, need to filter out
+    //    so that we do not unnecessarily dump output logs
+    if ((throwable instanceof TestAbortedException) // opentest4j
+        || "AssumptionViolatedException".equals(throwable.getClass().getSimpleName()) // junit
```

Comment:
This may look odd but module does not have dependency on `junit`, and I am bit worried about OSGi providing different version of the class. Hence chose to check the name, looks explicit enough and works.